blog/_posts/2017-02-23-scalafix-v0.3.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: blog
+post-type: blog
+by: Ólafur Páll Geirsson
+title: "Refactor with scalafix v0.3"
+---
+
+I am happy to announce the release of [scalafix v0.3][scalafix].
+This release leverages the new scala.meta semantic API to provide a re-designed `Rewrite` and `Patch` API for refactorings.
+Let me explain what that means word by word.
+
+## scala.meta semantic API
+
+Scala.meta recently announced the [first release of its semantic API][meta-1.6].
+This release is the product of close collaboration for the past several months between
+[@xeno-by](https://twitter.com/xeno_by) at Twitter and myself at the Scala Center.
+The semantic API provides operations to query information from the compiler.
+
+The first version of the scala.meta semantic API makes it possible to query for the "symbol" of a name that appears in a Scala source file.
+A symbol is a unique identifier of a definition such as a class, val, def or trait.
+For example, the symbol of `println` from the standard library is `_root_.scala.Predef.println(Ljava/lang/Object;)V`.
+
+Symbols are created by "scalahost", a compiler plugin.
```

Comment:
I expanded this into "Scalahost is a compiler plugin in the scala.meta project that is responsible for extracting symbols from the compiler and mapping them to scala.meta syntax trees."

It all makes more sense now that I properly explained name resolution.